lib/client/language_tool_client.dart

```diff
@@ -38,7 +38,8 @@ class LanguageToolClient {
     );
 
     final languageToolAnswer = LanguageToolRaw.fromJson(
-      json.decode(result.body) as Map<String, dynamic>,
+      //json.decode(result.body) as Map<String, dynamic>,
+			json.decode(utf8.decode(result.bodyBytes)) as Map<String, dynamic>,
```

Comment:
No commented-out code in VCS, please.
```suggestion
			json.decode(utf8.decode(result.bodyBytes)) as Map<String, dynamic>,
```